E2E RP failed to start

```

COSMOSDB_ACCOUNT=e2einfra-eastus
DATABASE_NAME=root
RESOURCEGROUP=e2einfra-eastus

CLUSTER=v4-e2e-Ve261b903
ARO_RESOURCEGROUP=v4-e2e-rg-Ve261b903-eastus
CLUSTER_RESOURCEGROUP=aro-v4-e2e-rg-Ve261b903-eastus
KUBECONFIG=/data/vsts-agent/_work/2/s/gopath/src/github.com/Azure/ARO-RP/v4-e2e-Ve261b903.kubeconfig
CLUSTERSPN=/data/vsts-agent/_work/2/s/gopath/src/github.com/Azure/ARO-RP/v4-e2e-Ve261b903.json

PROXY_HOSTNAME=vm0.aroe2eproxy.eastus.cloudapp.azure.com
######################################
########## 🚀 Run ARO RP in background ##########
########## ？Checking ARO RP Status ##########
time="2020-04-27T03:29:44Z" level=info msg="starting, git commit e261b903" func="main.main()" file="cmd/aro/main.go:40"
time="2020-04-27T03:29:44Z" level=info msg="uuid 8211ec37-fc1b-405e-9db9-768808260a95" func="main.rp()" file="cmd/aro/rp.go:37"
time="2020-04-27T03:29:44Z" level=warning msg="running in development mode" func="env.NewEnv()" file="pkg/env/env.go:61"
Attempt 0 - local RP is NOT up. Code : 000, waiting
Attempt 1 - local RP is NOT up. Code : 000, waiting
Attempt 2 - local RP is NOT up. Code : 000, waiting
time="2020-04-27T03:29:49Z" level=info msg=listening func="main.rp()" file="cmd/aro/rp.go:118"
Attempt 9 - local RP is NOT up. Code : 500, waiting
The STDIO streams did not close within 10 seconds of the exit event from process '/bin/bash'. This may indicate a child process inherited the STDIO streams and has not yet exited.
##[error]Bash exited with code '1'.
Finishing: 🚀 Run ARO RP & E2E
```

https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=30601523&view=logs&j=10f66467-59e5-53bc-0c8a-42cc89770de4&t=cc311011-c8c8-5cb4-274c-19f32b3aecb1

Need more logs on RP starting to see why this failed.

--- a/pkg/frontend/ready_get.go
+++ b/pkg/frontend/ready_get.go
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
 )
 
 // checkReady checks the ready status of the frontend to make it consistent
@@ -15,8 +16,12 @@ import (
 // minutes before indicating health.  This ensures that there will be a gap in
 // our health metric if we crash or restart.
 func (f *frontend) checkReady() bool {
-	return time.Now().Sub(f.startTime) > 2*time.Minute &&
-		f.ready.Load().(bool) &&
+	if _, ok := f.env.(env.Dev); !ok &&
+		time.Now().Sub(f.startTime) < 2*time.Minute {
+		return false
+	}
+
+	return f.ready.Load().(bool) &&
 		f.env.ArmClientAuthorizer().IsReady() &&
 		f.env.AdminClientAuthorizer().IsReady()
 }

